coverage/index.html

```diff
@@ -64,7 +64,7 @@ <h1>
 </div><!-- /wrapper -->
```

Comment:
Make sure `/coverage/*` is in your .gitignore file and run `git rm coverage/index.html` in your terminal to remove this file from your git history